app/src/main/java/org/mifos/mobile/ui/beneficiary/presentation/IconAndText.kt

```diff
@@ -0,0 +1,90 @@
+package org.mifos.mobile.ui.beneficiary.presentation
+
+import android.content.res.Configuration
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+
+
+/**
+ * this is a reusable composable function that is made up of a text and icon composable
+ * some of the intake parameters are
+ * @param[icon]
+ * @param[icondescription]
+ * @param[text]
+ * @param[iconClick]
+ *
+ * */
+@Composable
+fun IconAndText(
```

Comment:
function name is not meaningful, please name it with what it does.

Reply:
done

Reply:
![Screenshot (118)](https://github.com/openMF/mifos-mobile/assets/90254566/9abc78b9-41dc-4ec6-adab-ff48a4d338ff)